dist/info/moonlight_libretro.info

```diff
@@ -0,0 +1,23 @@
+## All data is optional, but helps improve user experience.
+
```

Comment:
Don't think we need this header here :wink: 
``` suggestion
```

Reply:
updated.